common/scripts/setup_env.sh

```diff
@@ -26,13 +26,13 @@ set -e
 LOCAL_ARCH=$(uname -m)
 export LOCAL_ARCH
```

Comment:
I spent a good 5 minutes staring at this change to understand what you were after here :) You could simplify:

```
# Pass environment set target architecture to build system
if [[ -z ${TARGET_ARCH} ]]; then
    export TARGET_ARCH
elif ...
# Otherwise set TARGET_ARCH based upon LOCAL_ARCH
```
and keep lines 29-35 intact.

Reply:
Correct, I was working on a PR where this logic was still in the makefile so I was using the same logic that was in the original makefile where it would set the var otherwise use the default. 

I definitely like this suggestion though!